### ISSUE TITLE
Golden Dragon - Dragon Feat Issue 

### ISSUE THREAD

**Issue by SpencerMycek** [NosVladimir/KineticArchetypes#13]
Leaving an issue here and https://github.com/Vek17/TabletopTweaks-Base/issues/676

Having Both TTT-Base and KineticArchetypes active when leveling from Mythic 8-9, Golden Dragon path, when you click to the first "Dragon Feat" selection, the game will crash. Unfortunately, I can't see any logs in Player.Log or GameLogFull.txt that indicates why it breaks

Tested on v2.1.5r
Toybox 1.5.6
TTT-Base 2.5.14
TTT-Core 0.6.7
Kinetic Archetype 1.0.1

**Comment by NosVladimir** [comment]
Thanks for the bug report. I've identified the issue to be with my weapon training kinetic blade. For some reason it caused advanced weapon training to be listed as an option in its own selection list, potentially causing infinite loops.

I was not able to reproduce the crash with simply clicking on the Dragon Feat, but I reproduced it with another sequence of steps: click on one of the four advanced weapon training other than the first one, then from the dropdown list, click on advanced weapon training again, then the game will crash. If I turn off esoteric blade (hence not adding weapon training kinetic blade), there will be no advanced weapon training in the dropdown list, and the game won't crash.

This behaviour is the same with and without TTT enabled. I don't know whether it's potentially a TTT-related issue for crashing on simply clicking on Dragon Feat, but I'll first see if I can resolve the advanced weapon training infinite loop problem.
